tests/FakeItEasy.Specs/ConfigurationSpecs.cs

```diff
@@ -697,6 +697,7 @@ public virtual void DoSomething()
 
             public void DoSomethingNonVirtual()
             {
+                this.WasCalled = true;

```

Comment:
This method was causing CA1822, because it didn't reference `this` so it could be made static. But we don't actually want it to be static. Adding this makes the warning go away.